locations/spiders/claires.py

```diff
@@ -9,7 +9,7 @@ class ClairesSpider(RioSeoSpider):
     item_attributes = {"brand_wikidata": "Q2974996"}
     end_point = "https://maps.stores.claires.com/api/"
 
-    def post_process_feature(self, feature: Feature, location: dict) -> Iterable[Feature]:
```

Comment:
You don't need to make these diffs across every spider. If it's that important. In `RioSeoSpider` deprecate `post_process_feature`, and add `post_process_item` which calls `post_process_feature`.

Reply:
Well, right now this is a 100% refactoring rather than a trailing long term refactoring. If it were a few hundred spiders, I'd agree with your approach.

As far as I know, no rioseospiders are currently in progress; and if any do get added before this is merged, will maintain this branch.

When I looked at the reason this is named post_process_feature, rather than Item; it seemed that it was because it was written focused on the type Feature; rather than our type, Item.

So it seemed more sensible to align the APIs rather than do half now, half later.